js/tests/unit/modal.js

```diff
@@ -166,16 +166,15 @@ $(function () {
     assert.expect(3)
     var done = assert.async()
 
-    var div = $('<div id="modal-test"/>')
-    div
+    $('<div id="modal-test"/>')
       .on('shown.bs.modal', function () {
         assert.ok($('#modal-test').length, 'modal insterted into dom')
         assert.ok($('#modal-test').is(':visible'), 'modal visible')
-        div.trigger($.Event('keydown', { which: 27 }))
+        $('#modal-test').trigger($.Event('keydown', { which: 27 }))
```

Comment:
This is how it is 8/10 cases above.
On Apr 29, 2015 11:03 PM, "Chris Rebert" <notifications@github.com> wrote:

> In js/tests/unit/modal.js
> <https://github.com/twbs/bootstrap/pull/16392#discussion_r29374410>:
>
> >        .on('shown.bs.modal', function () {
> >          assert.ok($('#modal-test').length, 'modal insterted into dom')
> >          assert.ok($('#modal-test').is(':visible'), 'modal visible')
> > -        div.trigger($.Event('keydown', { which: 27 }))
> > +        $('#modal-test').trigger($.Event('keydown', { which: 27 }))
>
> [image: :-1:] I don't see how this is an improvement
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twbs/bootstrap/pull/16392/files#r29374410>.
>